appveyor.yml

```diff
@@ -28,10 +28,12 @@ install:
         }
     - ps: |
         # the legacy WMF4 image only has the old preview SDKs of dotnet
-        if (-not ((dotnet --version).StartsWith('2.1.4')))
+        $globalDotJson = Get-Content .\global.json -Raw | ConvertFrom-Json
+        $dotNetCoreSDKVersion = $globalDotJson.sdk.version
+        if (-not ((dotnet --version).StartsWith($dotNetCoreSDKVersion)))
```

Comment:
should we be specifying a specific version or just ensuring that the `dotnet` version is high enough?

Reply:
I don't have a strong opinion on this to be honest but if we regularly keep up with the latest SDKs we should not fall behind the versions in the Appveyor images any more. It's probably best to revisit when some of this code gets moved into build modules